src/components/Image.vue

```diff
@@ -1,14 +1,18 @@
 <template>
-  <img :src="imageUrl"/>
+  <img :src="imageUrl" v-if="imageUrl" />
+  <ion-skeleton-text v-else animated></ion-skeleton-text>
```

Comment:
We can use self-closing tag here.

Reply:
Improved code readability by using self-closing app.
Thank You sir for reviewing my code.